### ISSUE TITLE
Add support for figure and ficaption tags

### ISSUE THREAD

**Issue by aleray** [nephila/django-ckeditor-filebrowser-filer#23]
Hi,

Image2 plugins have that and it is very useful. Would there be a way to integrate that in your plugin?

thanks,

**Comment by yakky** [comment]
Yes. Unfortunately code is a bit convoluted, thus I can't estimate the effort it will take.
If anyone has more ckeditor foo, any help is welcome!
